.zenodo.json

```
@@ -201,16 +201,16 @@
       "name": "Thomas Guiot",
       "orcid": "0000-0001-7041-598X"
     },
-    {
-      "name": "Hanke, Michael",
-      "affiliation": "Institute of Neuroscience and Medicine, Brain & Behaviour (INM-7), Research Centre Jülich, Jülich, Germany and Institute of Systems Neuroscience, Medical Faculty, Heinrich Heine University Düsseldorf, Düsseldorf, Germany",
-      "orcid": "0000-0001-6398-6370"
-    },
     {
       "name": "Basavaraj, Arshitha",
       "affiliation": "Data Science and Sharing Team, National Institute of Mental Health, National Institutes of Health, Bethesda, MD, USA.",
       "orcid": "0000-0002-6984-7969"
     }
```

Just one missing comma:
```suggestion
    },
```

---

Thanks @adswa ! I generally just do `Commit Suggestion` but I appreciate you mentioning the equivalence between new commit and push from the fork. It's easy to forget that with one-click solutions. :)